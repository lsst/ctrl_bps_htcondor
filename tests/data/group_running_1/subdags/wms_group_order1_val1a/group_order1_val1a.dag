JOB label2_val1a_val2a "label2_val1a_val2a.sub" DIR "../../jobs/label2/val1a"
JOB label2_val1a_val2b "label2_val1a_val2b.sub" DIR "../../jobs/label2/val1a"
JOB label4_val1a_val2a "label4_val1a_val2a.sub" DIR "../../jobs/label4/val1a"
JOB label4_val1a_val2b "label4_val1a_val2b.sub" DIR "../../jobs/label4/val1a"
PARENT label2_val1a_val2a CHILD label4_val1a_val2a
PARENT label2_val1a_val2b CHILD label4_val1a_val2b
DOT group_order1_val1a.dot
NODE_STATUS_FILE group_order1_val1a.node_status
SET_JOB_ATTR bps_run_quanta= "label2:2;label4:2"
SET_JOB_ATTR bps_job_summary= "label4:2;label2:2"
