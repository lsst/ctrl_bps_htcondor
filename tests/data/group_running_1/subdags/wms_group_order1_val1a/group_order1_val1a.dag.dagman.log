000 (10108.000.000) 2025-03-10 11:06:23 Job submitted from host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
    DAG Node: wms_group_order1_val1a
...
001 (10108.000.000) 2025-03-10 11:06:24 Job executing on host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
...
005 (10108.000.000) 2025-03-10 11:06:59 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
...
