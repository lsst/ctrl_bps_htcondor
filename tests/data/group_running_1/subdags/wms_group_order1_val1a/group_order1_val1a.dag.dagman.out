03/10/25 11:06:24 ******************************************************
03/10/25 11:06:24 ** condor_scheduniv_exec.10108.0 (CONDOR_DAGMAN) STARTING UP
03/10/25 11:06:24 ** /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman
03/10/25 11:06:24 ** SubsystemInfo: name=DAGMAN type=DAGMAN(9) class=CLIENT(2)
03/10/25 11:06:24 ** Configuration: subsystem:DAGMAN local:<NONE> class:CLIENT
03/10/25 11:06:24 ** $CondorVersion: 24.0.4 2025-02-10 $
03/10/25 11:06:24 ** $CondorPlatform: X86_64-AlmaLinux_9.5 $
03/10/25 11:06:24 ** PID = 18322 RealUID = 27031
03/10/25 11:06:24 ** Log last touched time unavailable (No such file or directory)
03/10/25 11:06:24 ******************************************************
03/10/25 11:06:24 Using config source: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/etc/condor/condor_config
03/10/25 11:06:24 Using local config sources: 
03/10/25 11:06:24    /etc/condor/condor_config
03/10/25 11:06:24    /etc/condor/config.d/00-minicondor
03/10/25 11:06:24    /etc/condor/config.d/00-security
03/10/25 11:06:24    /etc/condor/config.d/10-stash-plugin.conf
03/10/25 11:06:24    /etc/condor/config.d/99-lsst
03/10/25 11:06:24    /etc/condor/condor_config.local
03/10/25 11:06:24    /home/testuser/.condor/user_config
03/10/25 11:06:24 config Macros = 99, Sorted = 99, StringBytes = 3220, TablesBytes = 3652
03/10/25 11:06:24 CLASSAD_CACHING is ENABLED
03/10/25 11:06:24 Daemon Log is logging: D_ALWAYS D_ERROR D_STATUS
03/10/25 11:06:24 DaemonCore: No command port requested.
03/10/25 11:06:24 DAGMAN_USE_STRICT setting: 1
03/10/25 11:06:24 DAGMAN_VERBOSITY setting: 3
03/10/25 11:06:24 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
03/10/25 11:06:24 DAGMAN_DEBUG_CACHE_ENABLE setting: False
03/10/25 11:06:24 DAGMAN_SUBMIT_DELAY setting: 0
03/10/25 11:06:24 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
03/10/25 11:06:24 DAGMAN_STARTUP_CYCLE_DETECT setting: False
03/10/25 11:06:24 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 100
03/10/25 11:06:24 DAGMAN_AGGRESSIVE_SUBMIT setting: False
03/10/25 11:06:24 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 1
03/10/25 11:06:24 DAGMAN_QUEUE_UPDATE_INTERVAL setting: 30
03/10/25 11:06:24 DAGMAN_DEFAULT_PRIORITY setting: 0
03/10/25 11:06:24 DAGMAN_SUPPRESS_NOTIFICATION setting: False
03/10/25 11:06:24 allow_events (DAGMAN_ALLOW_EVENTS) setting: 114
03/10/25 11:06:24 DAGMAN_RETRY_SUBMIT_FIRST setting: True
03/10/25 11:06:24 DAGMAN_RETRY_NODE_FIRST setting: False
03/10/25 11:06:24 DAGMAN_MAX_JOBS_IDLE setting: 1000
03/10/25 11:06:24 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
03/10/25 11:06:24 DAGMAN_MAX_PRE_SCRIPTS setting: 20
03/10/25 11:06:24 DAGMAN_MAX_POST_SCRIPTS setting: 20
03/10/25 11:06:24 DAGMAN_MAX_HOLD_SCRIPTS setting: 20
03/10/25 11:06:24 DAGMAN_MUNGE_NODE_NAMES setting: True
03/10/25 11:06:24 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
03/10/25 11:06:24 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
03/10/25 11:06:24 DAGMAN_ALWAYS_RUN_POST setting: False
03/10/25 11:06:24 DAGMAN_CONDOR_SUBMIT_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_submit
03/10/25 11:06:24 DAGMAN_CONDOR_RM_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_rm
03/10/25 11:06:24 DAGMAN_USE_DIRECT_SUBMIT setting: True
03/10/25 11:06:24 DAGMAN_PRODUCE_JOB_CREDENTIALS setting: True
03/10/25 11:06:24 DAGMAN_DEFAULT_APPEND_VARS setting: False
03/10/25 11:06:24 DAGMAN_INHERIT_ATTRS_PREFIX setting: 
03/10/25 11:06:24 DAGMAN_INHERIT_ATTRS setting: 
03/10/25 11:06:24 DAGMAN_ABORT_DUPLICATES setting: True
03/10/25 11:06:24 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
03/10/25 11:06:24 DAGMAN_PENDING_REPORT_INTERVAL setting: 60
03/10/25 11:06:24 DAGMAN_CHECK_QUEUE_INTERVAL setting: 28800
03/10/25 11:06:24 DAGMAN_AUTO_RESCUE setting: True
03/10/25 11:06:24 DAGMAN_MAX_RESCUE_NUM setting: 100
03/10/25 11:06:24 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
03/10/25 11:06:24 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
03/10/25 11:06:24 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
03/10/25 11:06:24 DAGMAN_MAX_JOB_HOLDS setting: 100
03/10/25 11:06:24 DAGMAN_HOLD_CLAIM_TIME setting: 20
03/10/25 11:06:24 ALL_DEBUG setting: D_FULLDEBUG
03/10/25 11:06:24 DAGMAN_SUPPRESS_JOB_LOGS setting: False
03/10/25 11:06:24 DAGMAN_REMOVE_NODE_JOBS setting: True
03/10/25 11:06:24 DAGMAN_REMOVE_JOBS_AFTER_LIMIT_CHANGE setting: False
03/10/25 11:06:24 DAGMAN will adjust edges after parsing
03/10/25 11:06:24 argv[0] == "condor_scheduniv_exec.10108.0"
03/10/25 11:06:24 argv[1] == "-Lockfile"
03/10/25 11:06:24 argv[2] == "group_order1_val1a.dag.lock"
03/10/25 11:06:24 argv[3] == "-Dag"
03/10/25 11:06:24 argv[4] == "group_order1_val1a.dag"
03/10/25 11:06:24 argv[5] == "-MaxIdle"
03/10/25 11:06:24 argv[6] == "1000"
03/10/25 11:06:24 argv[7] == "-CsdVersion"
03/10/25 11:06:24 argv[8] == "$CondorVersion: 24.0.4 2025-02-10 $"
03/10/25 11:06:24 argv[9] == "-dagman"
03/10/25 11:06:24 argv[10] == "/work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman"
03/10/25 11:06:24 argv[11] == "-AutoRescue"
03/10/25 11:06:24 argv[12] == "1"
03/10/25 11:06:24 argv[13] == "-DoRescueFrom"
03/10/25 11:06:24 argv[14] == "0"
03/10/25 11:06:24 argv[15] == "-do_recurse"
03/10/25 11:06:24 argv[16] == "-update_submit"
03/10/25 11:06:24 Workflow batch-id: <10093.0>
03/10/25 11:06:24 Workflow batch-name: <u_testuser_DM-46294_group_fail_20250310T160455Z.dag+10093>
03/10/25 11:06:24 Workflow accounting_group: <>
03/10/25 11:06:24 Workflow accounting_group_user: <>
03/10/25 11:06:24 DAGMAN_LOG_ON_NFS_IS_ERROR setting: False
03/10/25 11:06:24 Default node log file is: </work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1a/./group_order1_val1a.dag.nodes.log>
03/10/25 11:06:24 DAG Lockfile will be written to group_order1_val1a.dag.lock
03/10/25 11:06:24 DAG Input file is group_order1_val1a.dag
03/10/25 11:06:24 Parsing 1 dagfiles
03/10/25 11:06:24 Parsing group_order1_val1a.dag ...
03/10/25 11:06:24 Adjusting edges
03/10/25 11:06:24 Dag contains 4 total nodes
03/10/25 11:06:24 Bootstrapping...
03/10/25 11:06:24 Number of pre-completed nodes: 0
03/10/25 11:06:24 MultiLogFiles: truncating log file /work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1a/./group_order1_val1a.dag.nodes.log
03/10/25 11:06:24 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:24 Of 4 nodes total:
03/10/25 11:06:24  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:24   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:24     0       0        0       0       2          2        0        0
03/10/25 11:06:24 0 job proc(s) currently held
03/10/25 11:06:24 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeCount = 0.0; SubmitCycleTimeSum = 0.0; ]
03/10/25 11:06:24 Registering condor_event_timer...
03/10/25 11:06:25 Submitting HTCondor Node label2_val1a_val2a job(s)...
03/10/25 11:06:25 Submitting node label2_val1a_val2a from file label2_val1a_val2a.sub using direct job submission
03/10/25 11:06:25 	assigned HTCondor ID (10109.0.0)
03/10/25 11:06:25 Submitting HTCondor Node label2_val1a_val2b job(s)...
03/10/25 11:06:25 Submitting node label2_val1a_val2b from file label2_val1a_val2b.sub using direct job submission
03/10/25 11:06:25 	assigned HTCondor ID (10110.0.0)
03/10/25 11:06:25 Just submitted 2 jobs this cycle...
03/10/25 11:06:25 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:25 Of 4 nodes total:
03/10/25 11:06:25  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:25   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:25     0       0        2       0       0          2        0        0
03/10/25 11:06:25 0 job proc(s) currently held
03/10/25 11:06:25 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeAvg = 0.1120939254760742; SubmitCycleTimeCount = 1.0; SubmitCycleTimeMax = 0.1120939254760742; SubmitCycleTimeMin = 0.1120939254760742; SubmitCycleTimeStd = 0.1120939254760742; SubmitCycleTimeSum = 0.1120939254760742; ]
03/10/25 11:06:26 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:26 Reassigning the id of job label2_val1a_val2a from (10109.0.0) to (10109.0.0)
03/10/25 11:06:26 Event: ULOG_SUBMIT for HTCondor Node label2_val1a_val2a (10109.0.0) {03/10/25 11:06:25}
03/10/25 11:06:26 Number of idle job procs: 1
03/10/25 11:06:26 Reassigning the id of job label2_val1a_val2b from (10110.0.0) to (10110.0.0)
03/10/25 11:06:26 Event: ULOG_SUBMIT for HTCondor Node label2_val1a_val2b (10110.0.0) {03/10/25 11:06:25}
03/10/25 11:06:26 Number of idle job procs: 2
03/10/25 11:06:31 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:31 Event: ULOG_EXECUTE for HTCondor Node label2_val1a_val2a (10109.0.0) {03/10/25 11:06:31}
03/10/25 11:06:31 Number of idle job procs: 1
03/10/25 11:06:34 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:34 Event: ULOG_EXECUTE for HTCondor Node label2_val1a_val2b (10110.0.0) {03/10/25 11:06:34}
03/10/25 11:06:34 Number of idle job procs: 0
03/10/25 11:06:43 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:43 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1a_val2a (10109.0.0) {03/10/25 11:06:43}
03/10/25 11:06:43 Number of idle job procs: 0
03/10/25 11:06:43 Node label2_val1a_val2a job proc (10109.0.0) completed successfully.
03/10/25 11:06:43 Node label2_val1a_val2a job completed
03/10/25 11:06:43 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:43 Of 4 nodes total:
03/10/25 11:06:43  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:43   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:43     1       0        1       0       1          1        0        0
03/10/25 11:06:43 0 job proc(s) currently held
03/10/25 11:06:43 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.0090688599480523; EventCycleTimeCount = 18.0; EventCycleTimeMax = 0.1124329566955566; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02834867076006039; EventCycleTimeSum = 0.1632394790649414; LogProcessCycleTimeAvg = 0.01269453763961792; LogProcessCycleTimeCount = 4.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 0.0001740455627441406; LogProcessCycleTimeStd = 0.0248611537108944; LogProcessCycleTimeSum = 0.05077815055847168; SleepCycleTimeAvg = 1.000946190622118; SleepCycleTimeCount = 18.0; SleepCycleTimeMax = 1.001070022583008; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0002285155609427967; SleepCycleTimeSum = 18.01703143119812; SubmitCycleTimeAvg = 0.00590221505416067; SubmitCycleTimeCount = 19.0; SubmitCycleTimeMax = 0.1120939254760742; SubmitCycleTimeMin = 1.9073486328125E-06; SubmitCycleTimeStd = 0.02571549636272376; SubmitCycleTimeSum = 0.1121420860290527; ]
03/10/25 11:06:45 Submitting HTCondor Node label4_val1a_val2a job(s)...
03/10/25 11:06:45 Submitting node label4_val1a_val2a from file label4_val1a_val2a.sub using direct job submission
03/10/25 11:06:45 	assigned HTCondor ID (10111.0.0)
03/10/25 11:06:45 Just submitted 1 job this cycle...
03/10/25 11:06:45 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:45 Of 4 nodes total:
03/10/25 11:06:45  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:45   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:45     1       0        2       0       0          1        0        0
03/10/25 11:06:45 0 job proc(s) currently held
03/10/25 11:06:45 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.008617125059428968; EventCycleTimeCount = 19.0; EventCycleTimeMax = 0.1124329566955566; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02762023342863604; EventCycleTimeSum = 0.1637253761291504; LogProcessCycleTimeAvg = 0.01269453763961792; LogProcessCycleTimeCount = 4.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 0.0001740455627441406; LogProcessCycleTimeStd = 0.0248611537108944; LogProcessCycleTimeSum = 0.05077815055847168; SleepCycleTimeAvg = 1.000951817161158; SleepCycleTimeCount = 19.0; SleepCycleTimeMax = 1.001070022583008; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0002234273636391612; SleepCycleTimeSum = 19.01808452606201; SubmitCycleTimeAvg = 0.009413957595825195; SubmitCycleTimeCount = 20.0; SubmitCycleTimeMax = 0.1120939254760742; SubmitCycleTimeMin = 1.9073486328125E-06; SubmitCycleTimeStd = 0.02954875464047646; SubmitCycleTimeSum = 0.1882791519165039; ]
03/10/25 11:06:46 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:46 Reassigning the id of job label4_val1a_val2a from (10111.0.0) to (10111.0.0)
03/10/25 11:06:46 Event: ULOG_SUBMIT for HTCondor Node label4_val1a_val2a (10111.0.0) {03/10/25 11:06:45}
03/10/25 11:06:46 Number of idle job procs: 1
03/10/25 11:06:47 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:47 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1a_val2b (10110.0.0) {03/10/25 11:06:46}
03/10/25 11:06:47 Number of idle job procs: 1
03/10/25 11:06:47 Node label2_val1a_val2b job proc (10110.0.0) completed successfully.
03/10/25 11:06:47 Node label2_val1a_val2b job completed
03/10/25 11:06:47 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:47 Of 4 nodes total:
03/10/25 11:06:47  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:47   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:47     2       0        1       0       1          0        0        0
03/10/25 11:06:47 0 job proc(s) currently held
03/10/25 11:06:47 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01143877846854074; EventCycleTimeCount = 21.0; EventCycleTimeMax = 0.1124329566955566; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.03018223391974799; EventCycleTimeSum = 0.2402143478393555; LogProcessCycleTimeAvg = 0.008542497952779135; LogProcessCycleTimeCount = 6.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 0.0001559257507324219; LogProcessCycleTimeStd = 0.02030328891769327; LogProcessCycleTimeSum = 0.0512549877166748; SleepCycleTimeAvg = 1.000963790076119; SleepCycleTimeCount = 21.0; SleepCycleTimeMax = 1.00108814239502; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0002153345948995629; SleepCycleTimeSum = 21.02023959159851; SubmitCycleTimeAvg = 0.008558370850302956; SubmitCycleTimeCount = 22.0; SubmitCycleTimeMax = 0.1120939254760742; SubmitCycleTimeMin = 1.9073486328125E-06; SubmitCycleTimeStd = 0.02824256785958959; SubmitCycleTimeSum = 0.188284158706665; ]
03/10/25 11:06:48 Submitting HTCondor Node label4_val1a_val2b job(s)...
03/10/25 11:06:48 Submitting node label4_val1a_val2b from file label4_val1a_val2b.sub using direct job submission
03/10/25 11:06:48 	assigned HTCondor ID (10112.0.0)
03/10/25 11:06:48 Just submitted 1 job this cycle...
03/10/25 11:06:48 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:48 Event: ULOG_EXECUTE for HTCondor Node label4_val1a_val2a (10111.0.0) {03/10/25 11:06:47}
03/10/25 11:06:48 Number of idle job procs: 0
03/10/25 11:06:48 Reassigning the id of job label4_val1a_val2b from (10112.0.0) to (10112.0.0)
03/10/25 11:06:48 Event: ULOG_SUBMIT for HTCondor Node label4_val1a_val2b (10112.0.0) {03/10/25 11:06:48}
03/10/25 11:06:48 Number of idle job procs: 1
03/10/25 11:06:48 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:48 Of 4 nodes total:
03/10/25 11:06:48  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:48   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:48     2       0        2       0       0          0        0        0
03/10/25 11:06:48 0 job proc(s) currently held
03/10/25 11:06:48 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01094019412994385; EventCycleTimeCount = 22.0; EventCycleTimeMax = 0.1124329566955566; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02954753376063438; EventCycleTimeSum = 0.2406842708587646; LogProcessCycleTimeAvg = 0.007351977484566825; LogProcessCycleTimeCount = 7.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 0.0001559257507324219; LogProcessCycleTimeStd = 0.01880002633755423; LogProcessCycleTimeSum = 0.05146384239196777; SleepCycleTimeAvg = 1.000968163663691; SleepCycleTimeCount = 22.0; SleepCycleTimeMax = 1.00108814239502; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0002111439379560825; SleepCycleTimeSum = 22.0212996006012; SubmitCycleTimeAvg = 0.01632431279058042; SubmitCycleTimeCount = 23.0; SubmitCycleTimeMax = 0.1871750354766846; SubmitCycleTimeMin = 1.9073486328125E-06; SubmitCycleTimeStd = 0.04635205284292496; SubmitCycleTimeSum = 0.3754591941833496; ]
03/10/25 11:06:49 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:50 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:50 Event: ULOG_EXECUTE for HTCondor Node label4_val1a_val2b (10112.0.0) {03/10/25 11:06:49}
03/10/25 11:06:50 Number of idle job procs: 0
03/10/25 11:06:57 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:57 Event: ULOG_JOB_TERMINATED for HTCondor Node label4_val1a_val2a (10111.0.0) {03/10/25 11:06:56}
03/10/25 11:06:57 Number of idle job procs: 0
03/10/25 11:06:57 Node label4_val1a_val2a job proc (10111.0.0) completed successfully.
03/10/25 11:06:57 Node label4_val1a_val2a job completed
03/10/25 11:06:57 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:57 Of 4 nodes total:
03/10/25 11:06:57  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:57   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:57     3       0        1       0       0          0        0        0
03/10/25 11:06:57 0 job proc(s) currently held
03/10/25 11:06:57 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01402098132717994; EventCycleTimeCount = 31.0; EventCycleTimeMax = 0.1876089572906494; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.04086817089285284; EventCycleTimeSum = 0.4346504211425781; LogProcessCycleTimeAvg = 0.005184102058410645; LogProcessCycleTimeCount = 10.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 4.1961669921875E-05; LogProcessCycleTimeStd = 0.01574207406926701; LogProcessCycleTimeSum = 0.05184102058410645; SleepCycleTimeAvg = 1.000993274873303; SleepCycleTimeCount = 31.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0001820045371827592; SleepCycleTimeSum = 31.03079152107239; SubmitCycleTimeAvg = 0.01173374801874161; SubmitCycleTimeCount = 32.0; SubmitCycleTimeMax = 0.1871750354766846; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03975351803370635; SubmitCycleTimeSum = 0.3754799365997314; ]
03/10/25 11:06:59 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:06:59 Event: ULOG_JOB_TERMINATED for HTCondor Node label4_val1a_val2b (10112.0.0) {03/10/25 11:06:58}
03/10/25 11:06:59 Number of idle job procs: 0
03/10/25 11:06:59 Node label4_val1a_val2b job proc (10112.0.0) completed successfully.
03/10/25 11:06:59 Node label4_val1a_val2b job completed
03/10/25 11:06:59 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:59 Of 4 nodes total:
03/10/25 11:06:59  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:59   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:59     4       0        0       0       0          0        0        0
03/10/25 11:06:59 0 job proc(s) currently held
03/10/25 11:06:59 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01318224993619052; EventCycleTimeCount = 33.0; EventCycleTimeMax = 0.1876089572906494; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.03971228479340312; EventCycleTimeSum = 0.4350142478942871; LogProcessCycleTimeAvg = 0.004728642376986417; LogProcessCycleTimeCount = 11.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 4.1961669921875E-05; LogProcessCycleTimeStd = 0.01501044586737828; LogProcessCycleTimeSum = 0.05201506614685059; SleepCycleTimeAvg = 1.000997109846635; SleepCycleTimeCount = 33.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0001768912966571881; SleepCycleTimeSum = 33.03290462493896; SubmitCycleTimeAvg = 0.01104365376865163; SubmitCycleTimeCount = 34.0; SubmitCycleTimeMax = 0.1871750354766846; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03863178046321002; SubmitCycleTimeSum = 0.3754842281341553; ]
03/10/25 11:06:59 All jobs Completed!
03/10/25 11:06:59 Note: 0 total node deferrals because of -MaxJobs limit (0)
03/10/25 11:06:59 Note: 0 total node deferrals because of -MaxIdle limit (1000)
03/10/25 11:06:59 Note: 0 total node deferrals because of node category throttles
03/10/25 11:06:59 Note: 0 total PRE script deferrals because of -MaxPre limit (20) or DEFER
03/10/25 11:06:59 Note: 0 total POST script deferrals because of -MaxPost limit (20) or DEFER
03/10/25 11:06:59 Note: 0 total HOLD script deferrals because of -MaxHold limit (20) or DEFER
03/10/25 11:06:59 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:06:59 Of 4 nodes total:
03/10/25 11:06:59  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:06:59   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:06:59     4       0        0       0       0          0        0        0
03/10/25 11:06:59 0 job proc(s) currently held
03/10/25 11:06:59 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01318224993619052; EventCycleTimeCount = 33.0; EventCycleTimeMax = 0.1876089572906494; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.03971228479340312; EventCycleTimeSum = 0.4350142478942871; LogProcessCycleTimeAvg = 0.004728642376986417; LogProcessCycleTimeCount = 11.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 4.1961669921875E-05; LogProcessCycleTimeStd = 0.01501044586737828; LogProcessCycleTimeSum = 0.05201506614685059; SleepCycleTimeAvg = 1.000997109846635; SleepCycleTimeCount = 33.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0001768912966571881; SleepCycleTimeSum = 33.03290462493896; SubmitCycleTimeAvg = 0.01104365376865163; SubmitCycleTimeCount = 34.0; SubmitCycleTimeMax = 0.1871750354766846; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03863178046321002; SubmitCycleTimeSum = 0.3754842281341553; ]
03/10/25 11:06:59 Wrote metrics file group_order1_val1a.dag.metrics.
03/10/25 11:06:59 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01318224993619052; EventCycleTimeCount = 33.0; EventCycleTimeMax = 0.1876089572906494; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.03971228479340312; EventCycleTimeSum = 0.4350142478942871; LogProcessCycleTimeAvg = 0.004728642376986417; LogProcessCycleTimeCount = 11.0; LogProcessCycleTimeMax = 0.04998612403869629; LogProcessCycleTimeMin = 4.1961669921875E-05; LogProcessCycleTimeStd = 0.01501044586737828; LogProcessCycleTimeSum = 0.05201506614685059; SleepCycleTimeAvg = 1.000997109846635; SleepCycleTimeCount = 33.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.000303983688354; SleepCycleTimeStd = 0.0001768912966571881; SleepCycleTimeSum = 33.03290462493896; SubmitCycleTimeAvg = 0.01104365376865163; SubmitCycleTimeCount = 34.0; SubmitCycleTimeMax = 0.1871750354766846; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03863178046321002; SubmitCycleTimeSum = 0.3754842281341553; ]
03/10/25 11:06:59 **** condor_scheduniv_exec.10108.0 (condor_DAGMAN) pid 18322 EXITING WITH STATUS 0
